app/controllers/orders_controller.rb

```diff
@@ -51,4 +53,22 @@ def load_order
     flash[:danger] = t "order_not_found"
     redirect_to orders_path
   end
+
+  def update_order_table
+    t_ids = params[:order][:table_ids]
+    return if t_ids.empty?
+
+    t_ids.each do |t_id|
+      table = Table.find_by(id: t_id)
+      next if table.nil?
```

Comment:
```suggestion
      next if table.blank?
```
hạn chế xử dụng nil? trong rails nhé nó k check hết trường hơpj